src/components/SidePanel/SidePanel.js

```diff
@@ -219,7 +269,41 @@ Panel.propTypes = {
   compact: PropTypes.bool,
 }
 
+function useSidePanel() {
+  const value = useContext(SidePanelContext)
+  if (value === null) {
+    throw new Error(
+      'useSidePanel() need to be part of the SidePanel tree, ' +
```

Comment:
```suggestion
      'useSidePanel() was used without a SidePanel existing in the render tree' +
```
Perhaps this is more clear?

Reply:
I like that it describes the current situation rather than only saying what to do, but I would like to say that the hook need to be inside of the `SidePanel` render tree, not the other way.

What about this?

```suggestion
      'useSidePanel() was used outside of the SidePanel render tree, ' +
```

Reply:
I like that it describes the current situation rather than only saying what to do, but I would like to say that the hook need to be inside of the `SidePanel` render tree, not the other way.

What about this?

```suggestion
      'useSidePanel() was used outside of the SidePanel render tree, ' +
```

Reply:
Yes, let's use this!